src/communication/network_interface_cl.h

```diff
@@ -466,4 +466,7 @@ extern int mmon_disable_force ();
 /* tdes */
 extern void tdes_set_query_start_info (char *sql_user_text);
 extern void tdes_reset_query_start_info (PT_NODE * node);
+/* lob dir */
+extern int create_lob_dir (HFID * hfid, int *attrid_arr, int attrid_arr_length);
+extern int remove_lob_dir (HFID * hfid, int attrid);
```

Comment:
전반적인 코딩 컨벤션에 따라 공통 prefix를 적용하는게 어떨까요?
```suggestion
extern int lob_create_dir (HFID * hfid, int *attrid_arr, int attrid_arr_length);
extern int lob_remove_dir (HFID * hfid, int attrid);
```

Reply:
위 prefix 적용 시 NET_SERVER_CREATE_LOB_DIR, NET_SERVER_REMOVE_LOB_DIR 등도 일괄적으로 통일성 있게 적용 검토해 주세요.

Reply:
좋습니다. 적용했습니다.